src/main/groovy/com/excelsiorjet/gradle/plugin/ExcelsiorJetPlugin.groovy

```diff
@@ -95,19 +97,19 @@ class ExcelsiorJetPlugin implements Plugin<Project> {
                 return project.version.toString()
             }
         }
-        extension.conventionMapping.artifactName = { getArchiveName(project.tasks.getByPath(taskPath(project, "jar"))) }
+        extension.conventionMapping.artifactName = { getArchiveName(archiveTask) }
         if (!isWar) {
             extension.conventionMapping.mainJar = {
-                new File(project.tasks.getByPath(taskPath(project, "jar")).archivePath as String)
```

Comment:
This is not necessary. Plugin will always be applied to a specific project and such project is expected to have `jar` task.